test/e2e/suite/command/custom_header.go

```diff
@@ -87,6 +87,22 @@ var _ = Describe("Common registry users:", func() {
 				"-H", FoobarHeaderInput, "-H", AbHeaderInput).
 				MatchRequestHeaders(FoobarHeader, AbHeader).Exec()
 		})
+		It("copy --from-header", func() {
+			headerTestRepo := prepareHeaderTestRepo("from-header")
+			fromUrlPrefix := "http://localhost:5000/v2/" + ImageRepo
```

Comment:
can we user `://localhost:5000/v2/` here and replace the domain name with `Host`?

Reply:
Good advice. Changed.